integration_tests/testsuite/deploy_app.py

```diff
@@ -43,7 +51,26 @@ def _set_builder_image(builder):
                 fout.write(line.replace('${STAGING_BUILDER_IMAGE}', builder))
 
 
-def deploy_app(base_image, builder_image, appdir, yaml):
+def _record_latency_to_bigquery(deploy_latency, language):
+    current_date = datetime.datetime.now()
+    row = [(language, current_date, deploy_latency)]
+
+    project = os.environ.get(DEPLOY_LATENCY_PROJECT_ENV)
```

Comment:
This will just be None if the env var isn't set and then the bigquery will use the default project account which is set in kokoro/common.sh to create a client if project is None.